pkg/signup/service/signup_service.go

```diff
@@ -136,6 +133,11 @@ func (s *ServiceImpl) newUserSignup(ctx *gin.Context) (*toolchainv1alpha1.UserSi
 	return userSignup, nil
 }
 
+func isCRTAdmin(username string) bool {
+	newUsername := regexp.MustCompile("[^A-Za-z0-9]").ReplaceAllString(strings.Split(username, "@")[0], "-")
```

Comment:
Why do we need to do this replacement?

Reply:
yeah, its not needed to replace the special characters, but it doesn't harm to replace them. At first I just copied the first line of TransformUsername to get the string before @, but then kept the whole replace and regex for the sake of consistency.

Reply:
OK. I see... Both `@` split and transformation are probably redundant in the context of `-crtadmin` username checks but yeah.. I'm fine with keeping it.